src/pages/SignMessage/components/VerifySignatureForm.tsx

```diff
@@ -72,12 +72,14 @@ export const VerifySignatureForm = () => {
             <div className={styles.form}>
               <label className={styles.label}>Verify signature</label>
 
-              <textarea
+              <Field
+                component='textarea'
+                rows={10}
                 name='signedMessage'
                 value={values.signedMessage}
                 className={styles.field}
                 autoComplete='off'
-                onChange={(e) => {
+                onChange={(e: ChangeEvent<HTMLFormElement>) => {
```

Comment:
To mention that `VerifySignatureForm` and `VerifySignatureFormFields` components are not part of the refactoring. Just a few changes here to keep the PR as small as possbile